hack/deploy-hub-local/build-hub.sh

```diff
@@ -49,11 +50,21 @@ echo ">>>>>>>>>>>>>>>>>>>>>"
 if [ "${OC_DEPLOY_METAL}" = "yes" ]; then
     if [ "${OC_NET_CLASS}" = "ipv4" ]; then
         if [ "${OC_TYPE_ENV}" = "connected" ]; then
-            echo "Metal3 + Ipv4 + connected"
-            t=$(echo "${OC_RELEASE}" | awk -F: '{print $2}')
-            git pull
-            kcli create network --nodhcp --domain ztpfw -c 192.168.7.0/24 ztpfw
-            kcli create plan --force --paramfile=lab-metal3.yml -P disconnected="false" -P version="${VERSION}" -P tag="${t}" -P openshift_image="${OC_RELEASE}" -P cluster="${OC_CLUSTER_NAME}" "${OC_CLUSTER_NAME}"
+            if [ "${OC_ARCHITECTURE}" = "sno" ]; then
+		    echo "SNO + Metal3 + Ipv4 + connected"
+		    t=$(echo "${OC_RELEASE}" | awk -F: '{print $2}')
+		    git pull
+		    kcli create network --nodhcp --domain ztpfw -c 192.168.7.0/24 ztpfw
```

Comment:
instead of creating network, cluster and also dns entries below, you can put all of this in a plan file (which can then consume a single parameter file